src/ripple/core/Stoppable.h

```diff
@@ -97,9 +97,10 @@ class RootStoppable;
 
     7.  stop()
 
-        This first calls stopAsync(), and then blocks on each child Stoppable in
-        the in the tree from the bottom up, until the Stoppable indicates it has
-        stopped. This will usually be called from the main thread of execution
+        This first calls stopAsync(), then blocks on each child Stoppable in
```

Comment:
I wonder if it would be better if we extracted this massive comment into `Stoppable.md` under `src/ripple/core` and simply reference it from here.